pkgs/development/libraries/mesa/default.nix

```diff
@@ -23,12 +23,14 @@
 , libvdpau
 , llvmPackages
 , lm_sensors
+, mesa
```

Comment:
It's not bootstrapped, I'm using a native (for the build machine) build of Mesa to provide the intel-clc tool that is needed for cross builds.

Reply:
ah, that makes sense. thanks!

Reply:
> It's not bootstrapped, I'm using a native (for the build machine) build of Mesa to provide the intel-clc tool that is needed for cross builds.

How about adding the executable path into [`disallowedReferences`](https://nixos.org/manual/nix/stable/language/advanced-attributes#adv-attr-disallowedRequisites) to make sure that the `outPath`s are free from such references?

To avoid explicitly using the `.__spliced` attribute for such purpose, consider using `mesaNative` instead of `mesa` as a `<pkg>.override` argument, and specify `{ mesaNative = buildPackages.mesa; }` inside `all-packages.nix`.